docs/installation.rst

```diff
@@ -59,7 +59,7 @@ You can install CasperJS using `npm <http://npmjs.org/>`_::
 
 .. warning::
 
-   While CasperJS is installable via npm, :ref:`it's not a NodeJS package <faq_node>`, neither it's capable to require native NodeJS modules.
+   While CasperJS is installable via npm, :ref:`it is not a NodeJS module <faq_node>` and will not work with NodeJS out of the box.
 
```

Comment:
“and isn't capable of using a vast majority of NodeJS modules out there. Experiment and use your best judgement.”